src/view/pygame_environment.py

```diff
@@ -59,4 +73,4 @@ def _group_adaptation(self, car_group: Iterable[CarState]) -> Iterable[float]:
 
     def _car_adaptation(self, car_state: CarState) -> float:
         # TODO: something more sophisticated
-        return car_state.car.active_segment
+        return 2 * car_state.car.active_segment / (10 * car_state.car.ticks)
```

Comment:
Why do we need these 2 and 10?
```suggestion
        return car_state.car.active_segment / car_state.car.ticks
```

Reply:
Done